### PR TITLE
Feature/use new installation service

### PR DESCRIPTION
**Pull request by MWest2020**
I think this is it. 

Restructered the folders to `installation`
changed the `$id`'s in the schema
removed the functions. 

Don't know how to test it, though. Would like to know.

### CODE REVIEW

File: Installation/installation.json [hunk 1]
```diff
@@ -1,3 +1,68 @@
 {
-    "installationService": "CommonGateway\\KlantenBundle\\Service\\InstallationService"
+    "installationService": "CommonGateway\\KlantenBundle\\Service\\InstallationService",
+    "collections": [{
+        "reference": "https://vng.opencatalogi.nl/collection/Klanten.collection.json",
+        "schemaPrefix": "https://vng.opencatalogi.nl/schemas/brc"
```
Comment (WilcoLouwerse) [inline Installation/installation.json]: https://klantenBundle.commonground.nu/klant
Zie installationService->createCollections()

File: Installation/installation.json [hunk 2]
```diff
@@ -1,3 +1,68 @@
...
+    }],
+    "cards": {
```
Comment (WilcoLouwerse) [inline Installation/installation.json]: endpoints niet cards

File: Installation/installation.json [hunk 3]
```diff
@@ -1,3 +1,68 @@
...
+                "path": "subverblijfadressen",
+                "methods": []
+            }
+        ]
+    },
+    "endpoints": {
+        "schemas": [
+
+        ],
+        "subEndpoints": [
```
Comment (WilcoLouwerse) [inline Installation/installation.json]: Deze heb je niet nodig

File: Installation/installation.json [hunk 4]
```diff
@@ -1,3 +1,68 @@
...
+            },
+            {
+                "reference": "https://klantenBundle.commonground.nu/klant.subVerblijfBuitenland.schema.json",
+                "path": "subverblijfadressen",
+                "methods": []
+            }
+        ]
+    },
+    "endpoints": {
+        "schemas": [
```
Comment (WilcoLouwerse) [inline Installation/installation.json]: Alles vanuit cards mag hier heen en cards mag dan weg

File: composer.json [hunk 5]
```diff
@@ -16,7 +16,7 @@
     "minimum-stability": "dev",
     "require": {
         "php": ">=7.4",
-        "commongateway/corebundle": ">=0.0.1"
+        "commongateway/corebundle": ">=1.0.8"
```
Comment (WilcoLouwerse) [inline composer.json:19]: 👍

**Review by WilcoLouwerse — CHANGES_REQUESTED** [review]

File: Installation/installation.json (hunk 1 above)
Comment (MWest2020) [reply Installation/installation.json]: Like this, right? Kinda looked over it.

File: Installation/installation.json (hunk 2 above)
Comment (MWest2020) [reply Installation/installation.json]: yeah, bad copy paste into wrong section.

**Review by WilcoLouwerse — APPROVED** [review]

**Pull request merged**